### PR TITLE
fix: feed nav reacts to route changes

### PR DESCRIPTION
**Pull request by kopancek**
## Changes

Different approach to the fix, related to:
- https://github.com/dailydotdev/apps/pull/2960

### On those affected packages:
- [X] Have you done sanity checks in the webapp?
- [X] Have you done sanity checks in the extension?
- [X] Does this not break anything in companion?

### Did you test the modified components media queries?
- [X] MobileL (420px)
- [X] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-282 #done

### Preview domain
https://mi-282-feed-nav-fix2.preview.app.daily.dev

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]
This seems good from my tests

File: packages/shared/src/components/feeds/FeedNav.tsx [hunk 1]
```diff
@@ -28,12 +28,21 @@ enum FeedNavTab {
...
 }
 
+const urlToTab: Record<string, FeedNavTab> = {
+  '/': FeedNavTab.ForYou,
+  '/popular': FeedNavTab.Popular,
+  '/upvoted': FeedNavTab.MostUpvoted,
+  '/discussed': FeedNavTab.Discussions,
+  '/bookmarks': FeedNavTab.Bookmarks,
+  '/history': FeedNavTab.History,
+};
```
Comment (capJavert) [inline packages/shared/src/components/feeds/FeedNav.tsx:38]: We can probably use this in `<Tab url={} />` as well.

**Review by capJavert — APPROVED** [review]
Awesome 🤘

File: packages/shared/src/components/feeds/FeedNav.tsx (hunk 1 above)
Comment (kopancek) [reply packages/shared/src/components/feeds/FeedNav.tsx:38]: good idea, let me try :)

Comment (capJavert) [reply packages/shared/src/components/feeds/FeedNav.tsx:38]: On second though, you would need inverse of it. Non blocking from my end.

**Review by rebelchris — APPROVED** [review]

**Pull request merged**